src/device-interface.d.ts

```diff
@@ -44,7 +44,8 @@ interface InboundPMData {
 }
 
 interface TopContextData {
-     inputType: SupportedType
+     inputType: SupportedType,
+     credentials?: CredentialsObject[]
```

Comment:
this is the additional data we're sending up to the top frame

Reply:
You mean when generating passwords or when we want to store them?

Reply:
it only sends additional context (in our case a generated credential) up to the top frame so that it can be included in the tooltip data.